lib/lte_link_control/lte_lc.c

```diff
@@ -1012,10 +993,9 @@ int lte_lc_edrx_req(bool enable)
 {
 	int err;
 	int actt[] = {AT_CEDRXS_ACTT_WB, AT_CEDRXS_ACTT_NB};
-	char req[25];
 
 	if (!enable) {
-		err = at_cmd_write(edrx_disable, NULL, 0, NULL);
+		err = nrf_modem_at_printf(edrx_disable);
```

Comment:
Please mark that `nrf_modem_at_printf` can return positive errors on modem error, +CME error and +CMS error.  Suggestion is to log positive retval of `nrf_modem_at_printf` before returning -ENOEXEC.